### ISSUE TITLE
Some suggestions

### ISSUE THREAD

**Issue by ylazy** [bobaoapae/ane-urlloader#1]
Hi!

I would like to add some suggestions that I think it may be useful:

- Supporting events: When we want to add more than one callback.
- Error ID, bytesLoaded, bytesTotal should be included.
- License (MIT?) should be included.

Thank you for the nice library!

**Comment by bobaoapae** [comment]
Yes, nice ideas. I'm struggling right now to get this working for ios, found some issues on adt that Harman it's trying to fix, when this it's done I will start implementation new things and improving some parts. But the license it's a quick change, I will add it to the repository today.

Thanks.
